src/foam/nanos/approval/ApprovalRequestNotificationNotificationCitationView.js

```diff
@@ -60,45 +57,42 @@ foam.CLASS({
   methods: [
     function initE() {
       //this.SUPER();
-
       this.description = this.data.body;
       if ( this.description !== '' && this.description.length > 70 ) {
         this.description = this.description.substr(0, 70-1) + '...';
       }
 
       this
         .addClass(this.myClass())
-        .start()
-          // .start().addClass('monogram')
-          //   .add(this.monogram)
-          // .end()
-          .start().addClass('userSummaryDiv')
-            .start().addClass('userSummary')
-              .add(this.userSummary$)
+        .startContext({ mode: foam.u2.DisplayMode.RO, controllerMode: foam.u2.ControllerMode.VIEW })
```

Comment:
Is the controllerMode required?

Reply:
Yes, for some reason this view defaults to Write Mode. Eric, Danny and I did some testing with this yesterday and couldn't figure out why it did that.